src/main/java/com/simibubi/create/content/contraptions/components/crank/HandCrankBlock.java

```diff
@@ -70,6 +70,9 @@ public RenderShape getRenderShape(BlockState state) {
 	@Override
 	public InteractionResult use(BlockState state, Level worldIn, BlockPos pos, Player player, InteractionHand handIn,
 		BlockHitResult hit) {
+		if (player.isSpectator())
+			return InteractionResult.SUCCESS;
```

Comment:
```suggestion
			return InteractionResult.PASS;
```

Reply:
I thought about that too. I suppose it's kind of up to interpretation. However, since the base method returns `PASS` I think that's the best option.